modules/src/main/java/org/jboss/additional/testsuite/jdkall/present/jpa/hibernate/secondlevelcache/HibernateSecondLevelCacheTestCase.java

```diff
@@ -58,7 +58,7 @@
  * @author Scott Marlow (based on Madhumita's Hibernate test)
  */
 @RunWith(Arquillian.class)
-@EapAdditionalTestsuite({"modules/testcases/jdkAll/Wildfly/jpa/src/main/java"})
+@EapAdditionalTestsuite({"modules/testcases/jdkAll/Wildfly/jpa/src/main/java","modules/testcases/jdkAll/Eap7/jpa/src/main/java","modules/testcases/jdkAll/Eap71x-Proposed/jpa/src/main/java","modules/testcases/jdkAll/Eap71x/jpa/src/main/java"})
```

Comment:
Hello Jirka. The EAT for Eap7 server category tests https://github.com/jbossas/jboss-eap7/tree/7.x branch but for this branch I see no merge. (Last comment of https://github.com/jbossas/jboss-eap7/pull/2493). "modules/testcases/jdkAll/Eap7/jpa/src/main/java" should be removed.

Reply:
So it is enough to add just Eap71x things? Like this:
 +@EapAdditionalTestsuite({"modules/testcases/jdkAll/Wildfly/jpa/src/main/java","modules/testcases/jdkAll/Eap71x-Proposed/jpa/src/main/java","modules/testcases/jdkAll/Eap71x/jpa/src/main/java"})

Reply:
Yes. For now. There is a NEXT GENERATION AT STRUCTURE proposal that takes the AT STRUCTURES to the NEXT GENERATION for dynamically created software programs.